<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-02T16:44:49 GST" name="Suite" finished-at="2024-09-02T16:47:55 GST" duration-ms="186173">
    <groups>
    </groups>
    <test started-at="2024-09-02T16:44:49 GST" name="Submit Order Test" finished-at="2024-09-02T16:46:33 GST" duration-ms="103748">
      <class name="Framework.SubmitOrderTest">
        <test-method is-config="true" signature="luanchApplication()[pri:0, instance:Framework.SubmitOrderTest@5a18cd76]" started-at="2024-09-02T16:44:49 GST" name="luanchApplication" finished-at="2024-09-02T16:45:25 GST" duration-ms="35662" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- luanchApplication -->
        <test-method signature="submitOrder()[pri:0, instance:Framework.SubmitOrderTest@5a18cd76]" started-at="2024-09-02T16:45:25 GST" name="submitOrder" finished-at="2024-09-02T16:45:59 GST" duration-ms="34181" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Framework.SubmitOrderTest@5a18cd76]" started-at="2024-09-02T16:45:59 GST" name="teardown" finished-at="2024-09-02T16:45:59 GST" duration-ms="170" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="luanchApplication()[pri:0, instance:Framework.SubmitOrderTest@5a18cd76]" started-at="2024-09-02T16:45:59 GST" name="luanchApplication" finished-at="2024-09-02T16:46:32 GST" duration-ms="32517" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- luanchApplication -->
        <test-method signature="OrderHistoryTest()[pri:0, instance:Framework.SubmitOrderTest@5a18cd76]" started-at="2024-09-02T16:46:32 GST" name="OrderHistoryTest" finished-at="2024-09-02T16:46:33 GST" depends-on-methods="Framework.SubmitOrderTest.submitOrder" duration-ms="1049" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Framework.SubmitOrderTest@5a18cd76]" started-at="2024-09-02T16:46:33 GST" name="teardown" finished-at="2024-09-02T16:46:33 GST" duration-ms="86" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Framework.SubmitOrderTest -->
    </test> <!-- Submit Order Test -->
    <test started-at="2024-09-02T16:46:34 GST" name="Error Validations Test" finished-at="2024-09-02T16:47:55 GST" duration-ms="81740">
      <class name="Framework.ErrorValidationsTest">
        <test-method is-config="true" signature="luanchApplication()[pri:0, instance:Framework.ErrorValidationsTest@74c79fa2]" started-at="2024-09-02T16:46:34 GST" name="luanchApplication" finished-at="2024-09-02T16:47:04 GST" duration-ms="30740" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- luanchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:Framework.ErrorValidationsTest@74c79fa2]" started-at="2024-09-02T16:47:04 GST" name="LoginErrorValidation" finished-at="2024-09-02T16:47:06 GST" duration-ms="1274" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email or password..]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email or password..]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Framework.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Framework.ErrorValidationsTest@74c79fa2]" started-at="2024-09-02T16:47:06 GST" name="teardown" finished-at="2024-09-02T16:47:06 GST" duration-ms="115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="luanchApplication()[pri:0, instance:Framework.ErrorValidationsTest@74c79fa2]" started-at="2024-09-02T16:47:06 GST" name="luanchApplication" finished-at="2024-09-02T16:47:37 GST" duration-ms="31021" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- luanchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:Framework.ErrorValidationsTest@74c79fa2]" started-at="2024-09-02T16:47:37 GST" name="ProductErrorValidation" finished-at="2024-09-02T16:47:55 GST" duration-ms="17969" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Framework.ErrorValidationsTest@74c79fa2]" started-at="2024-09-02T16:47:55 GST" name="teardown" finished-at="2024-09-02T16:47:55 GST" duration-ms="127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Framework.ErrorValidationsTest -->
    </test> <!-- Error Validations Test -->
  </suite> <!-- Suite -->
</testng-results>
